packages/app-desktop/gui/ConfigScreen/controls/MissingPasswordHelpLink.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+import shim from '@joplin/lib/shim';
+import bridge from '../../../services/bridge';
+import StyledLink from '../../style/StyledLink';
+
+interface Props {
+	theme: any;
+	text: string;
+}
+
+const openMissingPasswordFAQ = () =>
+	bridge().openExternal('https://joplinapp.org/faq#why-did-my-sync-and-encryption-passwords-disappear-after-updating-joplin');
+
+// A link to a specific part of the FAQ related to passwords being cleared when upgrading
+// to a MacOS/ARM release.
+const MacOSMissingPasswordHelpLink: React.FunctionComponent<Props> = props => {
+	const macInfoLink = (
+		<StyledLink href="#"
+			onClick={openMissingPasswordFAQ}
+			style={props.theme.linkStyle}
+		>
+			{props.text}
+		</StyledLink>
+	);
+
+	// The FAQ section related to missing passwords is specific to MacOS/ARM -- only show it
+	// in that case.
+	const newArchitectureReleasedRecently = Date.now() <= Date.UTC(2023, 10); // 10 = November
+	const showMacInfoLink = shim.isMac() && process.arch === 'arm64' && newArchitectureReleasedRecently;
```

Comment:
> How about making it time limited - you display the message for two months from the release date (till 1 Nov), then you display the default?

I've done that for both help links (they currently use the same check).

Reply:
Thanks. I'm just thinking 2 months might be a bit short, so I've changed it to 3